django/forms/models.py

```diff
@@ -1214,8 +1214,7 @@ def _get_foreign_key(parent_model, model, fk_name=None, can_fail=False):
         fks_to_parent = [f for f in opts.fields if f.name == fk_name]
         if len(fks_to_parent) == 1:
             fk = fks_to_parent[0]
-            parent_list = parent_model._meta.get_parent_list()
-            parent_list.append(parent_model)
+            parent_list = [*parent_model._meta.parent_list, parent_model]
```

Comment:
Where you mentioned that this method:
> [...] _mutates_ the return list [...]

Did you mean the `.append` that was here, or does additional mutation occur somewhere in here and I'm just being blind to it? If it's the former, we could also do:
```python
parent_list = (*parent_model._meta.parent_list, parent_model)
```
to illustrate that, maybe? Doesn't make a blind lick of difference in the scheme of things, and I only bring it up precisely because you called out a _mutation_ occurring...